store/cli.go

```diff
@@ -0,0 +1,59 @@
+package store
+
+import (
+	"github.com/Layr-Labs/eigenda-proxy/store/precomputed_key/redis"
+	"github.com/Layr-Labs/eigenda-proxy/store/precomputed_key/s3"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	FallbackTargetsFlagName = withFlagPrefix("fallback-targets")
+	CacheTargetsFlagName    = withFlagPrefix("cache-targets")
+	ConcurrentWriteThreads  = withFlagPrefix("concurrent-write-routines")
+)
+
+func withFlagPrefix(s string) string {
+	return "store." + s
+}
+
+func withEnvPrefix(envPrefix, s string) []string {
+	return []string{envPrefix + "_STORE_" + s}
+}
```

Comment:
does singular `store`/`STORE` like this not make it sound like its talking about a single specific store? Just brianstorming here, but what about replacing with either `storage` or `stores` (considered `store_manager` but too long I think)

Reply:
Storage is probably the best fit term for short description of the component abstraction - will update!

Reply:
[3a6f3fb](https://github.com/Layr-Labs/eigenda-proxy/pull/195/commits/3a6f3fbd9f7ce92bbfe3b98871724bc7b3829a94)